packages/bot-web-ui/src/components/dashboard/tutorial-tab/common/search-input/index.tsx

```diff
@@ -2,35 +2,37 @@ import { observer } from '@deriv/stores';
 import { localize } from '@deriv/translations';
 import { useDBotStore } from 'Stores/useDBotStore';
 import React from 'react';
+import debounce from 'lodash.debounce';
 
+const DEBOUNCE_INTERVAL_TIME = 1000;
 const SearchInput = observer(({ faq_value, setFaqSearchContent, prev_active_tutorials }) => {
     const { dashboard } = useDBotStore();
-    const { setActiveTabTutorial } = dashboard;
     const input_ref = React.useRef(null);
+    const { setActiveTabTutorial, filterTuotrialTab } = dashboard;
 
+    const throttleChange = React.useCallback(
```

Comment:
Shouldn't it be a debounce function? Not throttle right?

Reply:
done ✅